en/development/testing.rst

```diff
@@ -23,6 +23,9 @@ To install PHPUnit with Composer:
 
 .. code-block:: bash
 
+    $ php composer.phar require --dev phpunit/phpunit:"^5.7|^6.0"
```

Comment:
This is not reflected here:
https://github.com/cakephp/cakephp/blob/3.4.0/composer.json#L34